indico/web/flask/util.py

```diff
@@ -257,15 +257,14 @@ def url_for(endpoint, *targets, **values):
         values.update(locator)
 
     static_site_mode = bool(ContextManager.get('offlineMode'))
-    values.setdefault('_external', static_site_mode)
```

Comment:
IMHO it's easier to mark the non-working links as external as we discover them rather than marking every link that is available in the offline site as not external.